### DISCUSSION TITLE
Sketchware Components,Blocks,Selectors store

### DISCUSSION THREAD

**Discussion by DevVigilante** [Sketchware-Pro/Sketchware-Pro#732] [General]
When I switched through Sketchware to Sketchware pro I faced many difficulties to download and setup blocks and components from YouTube because many of youtuber says "You need to download my app to download these sk blocks" and many of the app link are broken because those applications were removed by play store while many of YouTuber lock their block file in zip and says "You need to watch our video starting to end to get block password" that is very irritating that is why
There should be a file `sketchware files.md` in this repository and there we can download sketchware pro block,components,selectors menu and more like this.
In the last of `README.md` there should be a link named `Download sketchware additional files` that redirect users to `sketchware files.md` in the repository.

**Comment by DevVigilante** [comment]
And those block should be free to use to anyone without any copyright restrictions,Attributions and these blocks will be managed by only trusted people's or contributiors in discord community

**Reply by yousefdevloper** [reply]
Hello.i have the blocks and the components.I USE SK PRO.soo if you want it pls replay

**Reply by liajainal87-maker** [reply]
Where i can to get your block?

**Reply by yousefdevloper** [reply]
I have blocks. Widgets. Do you have any communication app

**Comment by PranavPurwar** [comment]
youtubers still would be doing that since really nobody can force them to use the in-app block store

**Reply by DevVigilante** [reply]
I am not saying to force them to stop I am just saying we can add blocks that areprovided by Sketchware team which will be free and required no attribution and when YouTuber provide blocks they write some attributes with them like in comments 
//made by aur....
You know but those blocks will be like official blocks every blocks will be checked by Sketchware pro team,Contributors

**Reply by PranavPurwar** [reply]
the team ain't so big for that. And server costs too

**Reply by DevVigilante** [reply]
> the team ain't so big for that. And server costs too

I said in the repository md file and I think it is free of cost

**Reply by PranavPurwar** [reply]
free of cost doesn't mean that you used it as file hosting. bluffing a repository with non-essential useless data is never a good idea

**Reply by DevVigilante** [reply]
Humm!
I think for that Sketchware pro should create new repository and host it in github pages and that's a good idea

**Reply by PranavPurwar** [reply]
that's not how github pages work

**Comment by yousefdevloper** [comment]
Hello.i have the blocks and the components.I USE SK PRO.soo if you want it pls replay

**Reply by danielclement921-tech** [reply]
O want it

**Reply by yousefdevloper** [reply]
I have blocks. Widgets. Do you have any communication app

**Comment by amargantenf5-cpu** [comment]
It's below: 
https://drive.google.com/file/d/1jYNYJnF0y2X73fvJzV_A5tkGm5L3MI8U/view?usp=drivesdk

**Reply by urfavxbf** [reply]
need request bro, pls let me in

**Reply by s84939563-create** [reply]
Please for access to link

**Comment by DevVigilante** [comment]
I was a total ediot back then.
(I will say the same after 1 year)

**Reply by liajainal87-maker** [reply]
the problem in sketchware is that it can't display youtube videos
